README.md

```diff
@@ -48,108 +49,104 @@ Remove all the completed todos after the `Clear completed` button click:
 - in case of any error show error message but process success deletions;
 
 ## If you want to enable tests
+
 - open `cypress/integration/page.spec.js`
 - replace `describe.skip` with `describe` for the root `describe`
 
 ## Instructions
+
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_todo-app-add-and-delete/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://Valbel93.github.io/react_todo-app-add-and-delete/) and add it to the PR description.
```

Comment:
Add this demo link to the PR description 
<img width="955" alt="image" src="https://github.com/user-attachments/assets/21a90281-ec56-4e20-ab25-73f078a0dd2d" />